webplugin/css/app/mck-sidebox-1.0.css

```diff
@@ -6372,4 +6347,79 @@ body.accesibility :focus {
 .mck-msg-left:last-child [data-hidepostformsubmit="true"] {
 	display: block;
 	visibility: visible;
+}
+
+/* ----------------- VoiceNote feature ------------- */
```

Comment:
can we move voice related code both css and js to a separate file? these 2 files are becoming larger and larger with new code, will be difficult to work on these

Reply:
Moved to different files

Reply:
saw the file km-voice-note.css but mck-sidebox-1.0.css is still showing new css for voice